src/DetectorConstruction.h

```diff
@@ -24,12 +24,13 @@ class DetectorConstruction: public G4VUserDetectorConstruction
   void ConstructWorld(G4VPhysicalVolume&);
   void ConstructWLSPlate(G4VPhysicalVolume*) const;
   void ConstructPhotosensors(G4VPhysicalVolume*) const;
+  void ConstructReflectiveFoils(G4VPhysicalVolume*) const;
 
   void Assert(G4VPhysicalVolume*, const G4String&) const;
 
 private:
   const G4double world_size_;
-  const G4double plate_width_, plate_thickn_, plate_length_;
+  const G4double plate_width_, plate_thickn_, plate_length_, foil_thickn_;
```

Comment:
Use different lines for the declarations related to different geometry elements.